.dockerignore

```diff
@@ -0,0 +1,15 @@
+node_modules
+*/node_modules
+*/*/node_modules
+*/*/*/node_modules
```

Comment:
I think that you can do `node_modules/` similar to the gitignore file

Reply:
I did, and it copied a lot of stuff, the minimal solution was the following:
`**/node_modules`